lib/transaction.js

```diff
@@ -383,9 +383,38 @@ function serializeTx(txObj) {
     serializedTx += _buf32.toString('hex');
     serializedTx += zbufferutils.getPushDataLength(o.script);
     serializedTx += o.script;
-  }); // Locktime
+  }); // Only handles forward transfers at the moment
 
-  _buf16.writeUInt16LE(txObj.locktime, 0);
+  if (txObj.version == -4) {
+    serializedTx += zbufferutils.numToVarInt(0); // Write every csw
+
+    serializedTx += zbufferutils.numToVarInt(0); // Write every vsc
+
+    serializedTx += zbufferutils.numToVarInt(txObj.vft_ccout.length); // Write every vft
+
+    for (var i = 0; i < txObj.vft_ccout.length; i++) {
+      var _buf32 = Buffer.alloc(8); // Satoshis
+
+
+      _buf32.writeUInt32LE(txObj.vft_ccout[i].value & -1, 0);
```

Comment:
Shouldn't be 64LEBN? https://github.com/HorizenOfficial/bitcore-lib-zen/blob/master/lib/transaction/sidechain.js#L447